docs/README.rst

```diff
@@ -13,11 +13,46 @@ After each commit on ``master``, documentation is updated and published to `Read
 Build
 -----
 
+It is not necessary to re-build this documentation while modifying LightGBM's source code.
+The HTML files generated using ``Sphinx`` are not checked into source control.
+However, you may want to build them locally during development to test changes.
+
+Docker
+^^^^^^
+
+The most reliable way to build the documentation locally is with Docker, using `the same images readthedocs uses <https://hub.docker.com/r/readthedocs/build>`_.
```

Comment:
Hmm... This link gives me 404 error. And Docker Hub says `readthedocs` organization has no repositories.

![image](https://user-images.githubusercontent.com/25141164/150244392-113fe193-4014-400d-ae90-67f215cbda59.png)

![image](https://user-images.githubusercontent.com/25141164/150244459-35696031-d382-4670-b2ee-0893fd26b103.png)


But I can see specific image in that repository. Maybe that repository is "hidden"?..

![image](https://user-images.githubusercontent.com/25141164/150244594-939fdb25-3b28-4e3d-9ef1-d96bf486b3cd.png)

https://hub.docker.com/layers/readthedocs/build/ubuntu-20.04-2021.09.23/images/sha256-09630e9fa6553d602756a73b63907475f59cb3667f073578f88b857693f3d359?context=explore

Reply:
Hmmm, I don't get a 404 when I go there in my browser.

![image](https://user-images.githubusercontent.com/7608904/150274894-54b78d3c-721a-496b-8ea4-9976181179db.png)

Or using `curl -I https://hub.docker.com/r/readthedocs/build`

![image](https://user-images.githubusercontent.com/7608904/150274982-f5f163ef-2d24-4ef1-8a18-a3dc5d2865b4.png)

Notice that in both cases, I'm not authenticated with DockerHub. So I'm pretty confident that that page is publicly-accessible.

Could you try again? Maybe Docker Hub had some temporary issues around the time you tried this link.

Reply:
Today everything is OK 😎